packages/app/src/SignInPage.tsx

```diff
@@ -11,7 +11,7 @@ export function SignInPage(): JSX.Element {
     <SignInForm
       onSuccess={() => navigate(searchParams.get('next') || '/')}
       onForgotPassword={() => navigate('/resetpassword')}
-      onRegister={() => navigate('/register')}
+      onRegister={process.env.REGISTER_ENABLED === 'false' ? undefined : () => navigate('/register')}
```

Comment:
Just to confirm, this could be disabled for the app by adding `REGISTER_ENABLED=false` to an optional `.env`? If so, should we add this to https://github.com/medplum/medplum/blob/main/packages/app/.env.defaults?

Reply:
Correct, I didn't want to change the default behavior and trigger a breaking version change.

Adding to defaults is a good idea.  I'll do that.  Thanks @tonylee80